pages/letushelp/prearraignment/index.tsx

```diff
@@ -0,0 +1,82 @@
+import Head from 'next/head'
+import { copy } from '../../../constants/prearraignment-data'
+import { siteTitle } from '../../../constants/copy'
+import Grid from '@material-ui/core/Grid'
+import useLanguage from '../../../hooks/useLanguage'
+import { Title, Paragraph } from '../../../ui'
+import { useStyles } from '../../../constants/materialStyles'
+const PreArraignmentPage = () => {
+  const { language } = useLanguage()
+  const classes = useStyles()
+  const activeCopy = copy[language]
+  return (
+    <>
+      <Head>
+        <title>{`${siteTitle} | ${activeCopy.title}`}</title>
+      </Head>
+      <div style={{ margin: '6rem 0 2rem 0' }}>
+        <Grid container spacing={3}>
+          <Grid item xs={2}>
+            <img
```

Comment:
can you introduce a media query to make these smaller on a view >700px or so? they are running off the edge and it looks a bit muddied.

Reply:
Ok added the query. I seen the mobile view was kinda messed up. Thanks for spotting that